src/components/CoinContainer/coincontainer.js

```diff
@@ -16,17 +16,14 @@ import {
   LoadMoreButton,
   TableNumberContainer,
   StyledLink,
+  SmallChartContainer,
 } from "./coincontainer.styles";
-import {
-  ProgressBar,
-  CoinChart,
-  CoinNumber,
-  AbbreviatedNumber,
-} from "components";
+import { ProgressBar, CoinNumber, AbbreviatedNumber, Chart } from "components";
 import chartReducer from "../../utils/ChartReducer";
 
-const DynamicRow = (props) => (
-  <>
+const DynamicRow = (props) => {
+  const theme = useTheme();
+  return (<>
     {props.list.map((element, index) => {
       return (
         <TableRow key={Math.random() * 100000000}>
```

Comment:
oiiii

no randomly generated index, i'm sure you can find something else to use for key